source/utilities.cc

```diff
@@ -104,6 +105,43 @@ namespace aspect
       return static_cast<bool>(ifile); // only in c++11 you can convert to bool directly
     }
 
+    void
+    load_and_distribute_file_content(const std::string &filename,
+                                     const MPI_Comm &comm,
+                                     std::stringstream &datastream)
+    {
+      // Reset datastream
+      datastream.str(std::string());
+
+      if (Utilities::MPI::this_mpi_process(comm) == 0)
+        {
+          std::ifstream filestream(filename);
+          AssertThrow (filestream,
+                       ExcMessage (std::string("Could not open file <") + filename + ">."));
+
+          // Read data from disk
+          filestream >> datastream.rdbuf();
```

Comment:
Me neither, but I did not want to use the C-style read functions (and did not want to search for the size of the file), and it is option (2) of the following reference, so it should be standard
http://www.cplusplus.com/reference/istream/istream/operator%3E%3E/